src/containers/Humanize.js

```diff
@@ -42,13 +56,30 @@ export function Humanize(props) {
 
 function formatInput(value, base, decimals, displaySizes) {
 	if (value === 0) return '0';
+	if (value < 0) return value;
```

Comment:
Well, you should support also negative values ...

Reply:
@ateska  updated to support also negative values